oap-server/server-starter/src/main/resources/ui-initialized-templates/pulsar/pulsar-cluster.json

```diff
@@ -0,0 +1,447 @@
+                  "metricMode": "Expression",
+                  "moved": false,
+                  "expressions": [
+                    "meter_pulsar_storage_logical_size.sum(['cluster','node'])/1024/1024"
+                  ],
+                  "typesOfMQE": [
+                    "TIME_SERIES_VALUES"
+                  ],
+                  "graph": {
+                    "type": "Line",
+                    "step": false,
+                    "smooth": false,
+                    "showSymbol": true,
+                    "showXAxis": true,
+                    "showYAxis": true
+                  },
+                  "widget": {
+                    "name": "storage_logical_size",
+                    "title": "Storage Logical Size"
+                  }
+                },
+                {
+                  "x": 0,
+                  "y": 40,
+                  "w": 12,
+                  "h": 10,
+                  "i": "17",
+                  "type": "Widget",
+                  "id": "16-0-17",
+                  "metricTypes": [],
+                  "metrics": [],
+                  "metricMode": "Expression",
+                  "moved": false,
+                  "expressions": [
+                    "meter_pulsar_storage_write_rate.sum(['cluster','node'])"
+                  ],
+                  "typesOfMQE": [
+                    "TIME_SERIES_VALUES"
+                  ],
+                  "graph": {
+                    "type": "Line",
+                    "step": false,
+                    "smooth": false,
+                    "showSymbol": true,
+                    "showXAxis": true,
+                    "showYAxis": true
+                  },
+                  "widget": {
+                    "name": "storage_write_rate",
+                    "title": "Storage Write Rate"
+                  }
+                },
+                {
+                  "x": 0,
+                  "y": 30,
+                  "w": 12,
+                  "h": 10,
+                  "i": "18",
+                  "type": "Widget",
+                  "id": "16-0-18",
+                  "metricTypes": [
+                    "readLabeledMetricsValues"
+                  ],
+                  "metrics": [
+                    "meter_pulsar_storage_size/1024/1024"
+                  ],
+                  "metricMode": "General",
+                  "moved": false,
+                  "expressions": [],
+                  "metricConfig": [],
+                  "graph": {
+                    "type": "Line",
+                    "step": false,
+                    "smooth": false,
+                    "showSymbol": true,
+                    "showXAxis": true,
+                    "showYAxis": true,
+                    "legend": {
+                      "asTable": false
+                    }
+                  },
+                  "widget": {
+                    "name": "storage_size",
+                    "title": "Storage Size"
```

Comment:
please add the unit of the storage size

Reply:
Should we add the metrics in the title?

Reply:
No, I mean need a `Unit` to indicate `MB/GB`